regparser/web/jobs/utils.py

```diff
@@ -1,7 +1,11 @@
 from regparser.tasks import run_eregs_command
+from django.conf import settings as django_settings
+from django.core.mail import get_connection, send_mail
 import django_rq
 import settings
 
+eregs_site_api_url = getattr(settings, "EREGS_SITE_API_URL")
```

Comment:
I think you can just use `settings.EREGS_SITE_API_URL` here, rather than the `getattr`.

I'm also not sure why this is a module constant rather than having users refer to the field in `settings` directly? If it's a run-time dependency check thing, would it make more sense to put in the application initializer? "You need to configure XYZ [dies]"?